src/pages/signin/TermsAndLicenses/index.native.js

```diff
@@ -1,3 +1,3 @@
-import TermsOnly from './TermsOnly';
+import TermsWithLicenses from './TermsWithLicenses';
 
-export default TermsOnly;
+export default TermsWithLicenses;
```

Comment:
Since we were previously only showing terms on mobile but now we are showing terms with licenses, could you update the tests of this PR to ensure that QA knows that on mobile we should be showing licenses on mobile?

Reply:
Do you mean adding this to the **QA Steps** in the PR description?

Reply:
Yes, that works fine as well 👍. The Tests section is for PR reviewer tests and the QA section is for QA, but most of the time they're the same